### PR TITLE
DBData.zip deleted. Now, data loads from web (Merge Second) (#54)

### LINKED ISSUE #54: Del DBData.zip from repository and make it downloadable by the script
**DBData.zip** is too big to work with it in the repository, so it will be better to put it to google.drive or other file-server and load it if necessary by the datadownload command.

### PR DESCRIPTION
**Pull request by Victurus**
close #54 
Main changes:
 * downloaddata script changed:
   - added two new functions for loading and unzipping
     data file
 * added .credentials/secretfile.json in order to provide
     the connection to google.drive where data was stored
 * DBData.zip was deleted.
Small changes:
 * requirements.txt updated (need for data loading)
 * architecture.md updated
 * README.md updated (the downloaddata command call changed)
 * .gitignore updated

### CODE REVIEW

**Comment by Victurus** [comment]
@Devkalion, Test this please on your computer before approving(cause you have linux like the second main system). Does the load starts?

File: bookshelf/datadownloader/management/commands/downloaddata.py [hunk 1]
```diff
@@ -82,9 +112,45 @@ def handle(self, *args, **options):
...
+
+        store = Storage(credential_path)
+        credentials = store.get()
+
+        http = credentials.authorize(httplib2.Http())
+        drive_service = discovery.build('drive', 'v3', http=http)
+
+        file_id = '1zRsyvLePWX2pRWiRdck9zms-0lScB5Vd'
+        request = drive_service.files().get_media(fileId=file_id)
+        fh = io.FileIO(pathToLoad, "wb")
```
Comment (Devkalion) [inline bookshelf/datadownloader/management/commands/downloaddata.py]: Error in this row
`FileNotFoundError: [Errno 2] No such file or directory: "*/bookshelf/media/DBData.zip"
`

Comment (Devkalion) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: This error fixes with command `mkdir bookshelf/media`

Comment (Victurus) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: @Devkalion, this is a very strange error. This file should not exist at the beginning. Was your media directory empty? have you installed the requirements?

Comment (Devkalion) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: It works. Need to add check that `media` directory exists.

Comment (Victurus) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: Yeiiii, :smiley_cat:. I'll correct this in the script.

Comment (Devkalion) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: Although I think we should add only archive and books images to gitignore, but not the whole media dir

Comment (Victurus) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: Why not? We don't have any files to trace in media folder and I think that they will not appear.

Comment (SelinaIrra) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: I use media folder for keeping files)))

Comment (Victurus) [reply bookshelf/datadownloader/management/commands/downloaddata.py]: Ok than. I'll correct this.

**Pull request merged**